server/util.go

```diff
@@ -223,7 +225,13 @@ func SetCmdServerContext(cmd *cobra.Command, serverCtx *Context) error {
 		cmdCtx = cmd.Context()
 	}
 
-	cmd.SetContext(context.WithValue(cmdCtx, ServerContextKey, serverCtx))
+	cmdCtx = context.WithValue(cmdCtx, ServerContextKey, serverCtx)
+	cmdCtx = context.WithValue(cmdCtx, corectx.ViperContextKey{}, serverCtx.Viper)
```

Comment:
Since we only use viper to get config in modules cmds, added both viper & logger into cmd context so its not break the tests. 
Wdut @julienrbrt is it an api breaking?